src/platform/forms-system/src/js/components/PersonalInformation/index.js

```diff
@@ -68,28 +78,38 @@ const personalInformationPage = ({
       CustomPage: props => (
         <PersonalInformation
           {...props}
-          config={personalInfoConfig}
+          config={config}
           dataAdapter={dataAdapter}
           errorMessage={errorMessage}
           contentBeforeButtons={contentBeforeButtons}
           contentAfterButtons={contentAfterButtons}
         >
+          {header && (
+            <PersonalInformationHeader>{header}</PersonalInformationHeader>
+          )}
           {cardHeader && (
             <PersonalInformationCardHeader>
               {cardHeader}
             </PersonalInformationCardHeader>
           )}
-          {header && (
-            <PersonalInformationHeader>{header}</PersonalInformationHeader>
-          )}
           {note && <PersonalInformationNote>{note}</PersonalInformationNote>}
           {footer && (
             <PersonalInformationFooter>{footer}</PersonalInformationFooter>
           )}
         </PersonalInformation>
       ),
-      CustomPageReview: null,
-      hideOnReview: true,
+      CustomPageReview: hideOnReview
+        ? null
+        : props => (
+            <PersonalInformationReview
+              {...props}
+              config={personalInfoConfig}
+              dataAdapter={dataAdapter}
+              title={title}
+            />
+          ),
+      hideOnReview,
+      depends,
```

Comment:
adding depends since there has been a request to support conditional inclusion of this page based on auth or other criteria.